clients/nodejs/validator-registration.js

```diff
@@ -0,0 +1,365 @@
+            process.exit(0);
+        }
+
+        // We are in validator configuration generation mode
+        // We need to generate the validator paramaters and exit the tool
+        console.log('Generating new validator paramaters... \n\n');
+
+        // First we generate the validator address parameters.
+        const validatorKeyPair =  NimiqPOS.KeyPair.generate();
+        const validatorAddress = validatorKeyPair.toAddress();
+
+        console.log('   Validator Account: ');
+        console.log('Address: ');
+        console.log(validatorAddress.toUserFriendlyAddress());
+        console.log('Public key: ');
+        console.log(validatorKeyPair.publicKey.toHex());
+        console.log('Private Key: ');
+        console.log(validatorKeyPair.privateKey.toHex());
+
+        // Now we generate the signing key.
+        const signingKeyPair =  NimiqPOS.KeyPair.generate();
+
+        console.log('\n   Signing key: ');
+        console.log('Public key: ');
+        console.log(signingKeyPair.publicKey.toHex());
+        console.log('Private key: ');
+        console.log(signingKeyPair.privateKey.toHex());
+
+        // Finally we generate the voting key (BLS)
+        const votingKeyPair = NimiqPOS.BLSKeyPair.generate();
+
+        console.log('\n   Voting BLS key: ');
+        console.log('Public key: ');
+        console.log(votingKeyPair.publicKey.toHex());
+        console.log('Secret key: ');
+        console.log(votingKeyPair.secretKey.toHex());
+
+        // Now we write the configuration to a JSON file
+        const validatorConfiguration = {
+            'ValidatorAccount': {
+                'Address': validatorAddress.toUserFriendlyAddress(),
+                'PublicKey': validatorKeyPair.publicKey.toHex(),
+                'PrivateKey': validatorKeyPair.privateKey.toHex(),
+            },
+            'SigningKey': {
+                'PublicKey': signingKeyPair.publicKey.toHex(),
+                'PrivateKey': signingKeyPair.privateKey.toHex(),
+            },
+            'VotingKey': {
+                'PublicKey': votingKeyPair.publicKey.toHex(),
+                'SecretKey': votingKeyPair.secretKey.toHex(),
+            }
+        };
+
+        // converting the JSON object to a string
+        const data = JSON.stringify(validatorConfiguration);
+
+        try {
+            fs.writeFileSync(pathToValidatorsFile, data);
+        } catch (error) {
+            // logging the error
+            console.error(error);
+
+            throw error;
+        }
+
+        console.log('\n\nValidator configuration file sucessfully written (validator-keys.json).');
+
+       process.exit(0);
+    }
+
+    if (argv.network){
+        config.network = argv.network
+    }
+
+    console.log(' Reading validator configuration.. ');
+    let validator_config = JSON5.parse(fs.readFileSync(argv.validator));
+
+    console.log(validator_config);
+
+    Nimiq.Log.i(TAG, `Nimiq NodeJS Client starting (network=${config.network}`
```

Comment:
```suggestion
    Nimiq.Log.i(TAG, `Nimiq NodeJS Client starting (network=${config.network})`
```

Reply:
Addressed in https://github.com/nimiq/validator-registration-tool/commit/f813a2c64b903bcc9fdc2e4aa4241464c055b7b2#diff-23931728b323120ef1e2accc6b9a61f7c036ce73b7d4ed19f9609d4a83fcddbeR235.